### PR TITLE
VBM-123: Change the title of "#Nova poshta" field to "Delivery data"

### PR DESCRIPTION
**Pull request by natata7**
Знімок екрана 2022-09-01 о 21 05 26

### CODE REVIEW

File: packages/mobile/src/localization/languages/en/screens.ts [hunk 1]
```diff
@@ -43,8 +43,8 @@ export const screens = {
     CITY: 'City',
     ZIP_CODE: 'ZIP Code',
     ZIP_CODE_HINT: 'Enter ZIP Code',
-    NOVA_POSHTA: '# Nova Poshta',
-    NOVA_POSHTA_HINT: 'Enter Nova Poshta office',
+    NOVA_POSHTA: 'Delivery data',
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/localization/languages/en/screens.ts:46]:
```suggestion
    DELIVERY_DATA: 'Delivery data',
```

File: packages/mobile/src/localization/languages/en/screens.ts [hunk 2]
```diff
@@ -43,8 +43,8 @@ export const screens = {
...
+    NOVA_POSHTA_HINT: 'Enter delivery address',
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/localization/languages/en/screens.ts:47]: DELIVERY_DATA_HINT

Comment (natata7) [reply packages/mobile/src/localization/languages/en/screens.ts]: updated

File: packages/mobile/src/localization/languages/en/screens.ts (hunk 1 above)
Comment (natata7) [reply packages/mobile/src/localization/languages/en/screens.ts]: updated

**Review by seal323 — APPROVED** [review]

**Review by AleksandrTermenzhy — APPROVED** [review]

**Pull request merged**